### ISSUE TITLE
Question about SVTR position_prior output: attn_map and text_logits shape/generation

### ISSUE THREAD

**Issue by w777-star** [SijieLiu518/SGENet#10]
Hello, thank you very much for sharing the SGENet code.

I am trying to reproduce the TextZoom experiments and have a small question about the SVTR-based `position_prior`.
I noticed in issue #1 that the missing PaddleOCR2Pytorch files were suggested to be taken from frotms/PaddleOCR2Pytorch. I checked that repository, but the SVTR implementation seems to return only the recognition output and does not expose the attn_map required by SGENet. Could you please clarify whether a modified SVTR version was used?
In `model/sgenet.py`, the code uses:

```python
attn_map, text_logits = self.position_prior(x_)

**Comment by w777-star** [comment]
However, when using the released PaddleOCR2Pytorch SVTR implementation, I can only obtain the recognition output/logits, and I am not sure how the attn_map is generated.
Could you please kindly clarify:
Was the SVTR model modified to output both attn_map and text_logits?
What is the expected shape of attn_map?
Is text_logits the final SVTR recognition logits with shape [B, 25, 37]?
Any guidance would be very helpful for reproducing your results.
Thank you very much for your time!
